libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -828,6 +837,21 @@ bool AP_Arming::mission_checks(bool report)
                 return false;
             }
           }
+        if ((_required_mission_items & MIS_ITEM_CHECK_NAV_DIST) && (_max_arm_dist_m > 0)) {
```

Comment:
Don't need the required-mission-item bit

Reply:
Then the check will be enabled by default (unless we change the distance default to zero). I thought a sensible distance default with an enable bit would be better, as it centralizes mission check enabling/disabling

Reply:
We decided on the call that we were happy to enable by default. So we can use a distance of 0 to disable and don't need the extra bit. Some large value on plane, 5km or something and a smaller distance on copter, maybe 1km.